src/wp-includes/theme.php

```diff
@@ -4203,6 +4203,20 @@ function wp_is_block_theme() {
 	return wp_get_theme()->is_block_theme();
 }
 
+/**
+ * Given an element name, returns a class name.
+ * Alias of WP_Theme_JSON::get_element_class_name.
+ *
+ * @since 6.1.0
+ *
+ * @param string $element The name of the element.
+ *
+ * @return string The name of the class.
+ */
+function wp_theme_element_class_name( $element ) {
```

Comment:
For consistency with the original method, and for consistent use of the active verb, I'd suggest renaming this alias to `wp_theme_get_element_class_name()`. 🙂

Reply:
Makes sense! That does have a nicer ring to it 😊